pkg/controller/gkenetworkparamset/gkenetworkparamset_controller.go

```diff
@@ -305,6 +314,11 @@ func (c *Controller) syncGNP(ctx context.Context, params *networkv1.GKENetworkPa
 		return nil
 	}
 
+	err = c.syncDefaultPodRanges(ctx, params)
```

Comment:
add a comment explaining why are we calling this here and what do we expect to happen

Reply:
Done.